lib/log.go

```diff
@@ -36,11 +37,8 @@ type LogParams struct {
 	Source string
 }
 
-// DatasetLogItem is a line item in a dataset logbook
-type DatasetLogItem = logbook.DatasetLogItem
-
 // Log returns the history of changes for a given dataset
-func (m *LogMethods) Log(params *LogParams, res *[]DatasetLogItem) error {
+func (m *LogMethods) Log(params *LogParams, res *[]dsref.VersionInfo) error {
```

Comment:
The big win of this PR is both `list` and `log` now actually return the same struct, which based on the comment above `VersionInfo` has always been the goal. 

Having a list of datasets & a dataset history be describable by the same thing without a lot of bending over backwards is very helpful IMHO